controllers/dexserver_controller.go

```diff
@@ -487,13 +482,13 @@ grpc:
   tlsClientCA: /etc/dex/mtls/ca.crt
   reflection: true
 connectors:
-- type: github
-  id: github
-  name: GitHub
+- type: ` + m.Spec.Connectors[0].Type + `
```

Comment:
@cdoan1 when you pull in the latest from main you'll need to convert the type to a string: `string(m.Spec.Connectors[0].Type)`